src/gateway/game.gateway.ts

```
@@ -51,51 +50,123 @@ export class GameGateway implements OnGatewayConnection, OnGatewayDisconnect {
-      `WebSocket makeMove received - GameID: ${data.gameId}, ClientID: ${client.id}, Move: ${JSON.stringify(data.dto)}`,
-    );
-    // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
-    const userId = client.data?.user?.sub as string;
-    if (!userId) {
-      throw new WsException('Unauthenticated socket');
-    }
-    await client.join(data.gameId);
-    const game = await this.gameService.makeMove(data.gameId, userId, data.dto);
-    // Broadcast updated game state to everyone in the room
-    this.server.to(game._id.toString()).emit('moveMade', game);
+@SubscribeMessage('makeMove')
```

ESLint

[error] Insert `··`

(prettier/prettier)